utils/automation/ScanDeps/Parsers.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+
+# SPDX-FileContributor: © Rajul Jha <rajuljha49@gmail.com>
+
+# SPDX-License-Identifier: GPL-2.0-only
+
+import requests
+import json
+from typing import Dict, Union
+
+
+class Parser:
+    """
+    Parser to classify each component based on it's type.
+    Ex: If purl is pkg:pypi/django@1.11.1,
+    it is a pypi package and should belong to python_components.
+    """
+    def __init__(self, sbom_file: str):
+        """
+        Initialize components list and load the sbom_data.
+        Args:
+            sbom_file: str | Path to sbom file
+        """
+        with open(sbom_file, 'r') as file:
+            self.sbom_data = json.load(file)
+        self.python_components = []
+        self.npm_components = []
+        self.php_components = []
+        self.unsupported_components = []
+    
+    def classify_components(self):
+        """
+        Classify components based on it's type
+        """
+        for component in self.sbom_data.get('components',[]):
+            purl = component.get('purl','')
+            type = self._extract_type(purl)
+
+            if type == 'pypi':
+                self.python_components.append(component)
+            # elif type == 'npm':
+            #     self.npm_components.append(component)
+            # elif type == 'composer':
+            #     self.php_components.append(component)
```

Comment:
Dont we need npm/php support for now? @GMishx ??

Reply:
We do, but this PR concerns only Python dependencies. Let's keep this pr limited only to it for easy review.